woocommerce/woocommerce-android
# Issue/2043 update stats banner

Fixes #2043 by changing the banner message design when stats v4 is not available. I decided to merge this into the `dark-mode` branch since it required some design changes which might be incompatible when dark mode feature is live.

#### Screenshots
##### When stats is available
<img src="https://user-images.githubusercontent.com/22608780/77619034-62e46b80-6f5d-11ea-8ba7-f3873c2ef6c8.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/77619041-65df5c00-6f5d-11ea-9613-ddd575e3e81f.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/77619038-6546c580-6f5d-11ea-8d4d-094ef94c90b4.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/77619042-6677f280-6f5d-11ea-9c26-a1a8052e5c3d.png" width="300"/> 

##### When stats is no longer available

<img src="https://user-images.githubusercontent.com/22608780/77619142-9921eb00-6f5d-11ea-9558-c45485cdb515.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/77619146-9c1cdb80-6f5d-11ea-87c0-d76d670b659c.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/77619144-9aebae80-6f5d-11ea-810b-dd5d5dcf1b21.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/77619151-9cb57200-6f5d-11ea-957d-b6e89e15078c.png" width="300"/>

#### To test
- Uninstall the app and install the changes from this PR.
- Login to a site with WooCommerce greater than 4.0 or WooCommerce Admin installed.
- Notice that after login, the Stats availability banner is displayed.
- Uninstall the WC-Admin from your site or downgrade the WooCommerce plugin to a version less than 4.0.
- Open the app again and notice that the Stats Reverted banner is displayed.
- Clicking on `Learn More` button opens the WooCommerce plugin page.
- Clicking on the `Dismiss` button dismisses the banner.

Update release notes:
